DashKit/Mocks/MockPodCommManager.swift

```diff
@@ -317,33 +317,41 @@ public class MockPodCommManager: PodCommManagerProtocol {
             return
         }
         
+        // SDK returns .invalidProgram if bolus attempt is made during suspension.
```

Comment:
@ps2 Just wondering...how are the changes in this file related to the pause icon color?

Reply:
The delay is from a comment Matt made while testing this ticket, that he didn't see the expected spinner. The .invalidProgram something I added when I was testing and noticed we could bolus while suspended, which didn't seem right. So I tested the actual PodSDK, and it returns this error.

Reply:
Both are just making the mock pod behave a little more like a real pod. No effects on actual shipped Loop.